app/build.gradle

```diff
@@ -149,6 +156,13 @@ dependencies {
     compile('com.crashlytics.sdk.android:crashlytics:2.6.2@aar') {
         transitive = true;
     }
+
+    testCompile "org.mockito:mockito-core:$mockito"
+    androidTestCompile "com.android.support.test.espresso:espresso-core:$espresso"
+    androidTestCompile "com.android.support.test.espresso:espresso-intents:$espresso"
+    androidTestCompile "com.android.support.test:rules:$testRunner"
+    androidTestCompile "com.android.support.test:runner:$testRunner"
+    androidTestCompile "com.android.support:support-v4:24.1.1" // 23.4.0 causes dependency error
```

Comment:
Created PR to fix this build error https://github.com/vilnius/tvarkau-vilniu/pull/69